docs/running.rst

```diff
@@ -1059,8 +1059,8 @@ Asynchronous Execution of Regression Checks
 -------------------------------------------
 
 From version `2.4 <https://github.com/eth-cscs/reframe/releases/tag/v2.4>`__, ReFrame supports asynchronous execution of regression tests.
-This execution policy can be enabled by passing the option ``--exec-policy=async`` to the command line.
-The default execution policy is ``serial`` which enforces a sequential execution of the selected regression tests.
+Since version `3.0 <https://github.com/eth-cscs/reframe/releases/tag/v3.0>`__, the asynchronous execution policy is the default one.
```

Comment:
This link will be dead until the actual 3.0 release happens. I suggest simply writing: "This execution policy is the default."